modules/apps/27-interchain-accounts/controller/keeper/msg_server_test.go

```diff
@@ -25,46 +27,46 @@ func (suite *KeeperTestSuite) TestRegisterInterchainAccount_MsgServer() {
 
 	testCases := []struct {
 		name     string
-		expPass  bool
+		expErr   error
```

Comment:
Nit: just for consistency with the other tests, we could just switch the order of the last two and have `expErr` last.